### PR TITLE
adding link to media contribution

### PR DESCRIPTION
**Pull request by truhacevkir**
Adding link to MediaContribution.

Is needed to map link for podcasts, media participations and other media contributions from Cristin.

### CODE REVIEW

File: nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java [hunk 1]
```diff
@@ -29,16 +31,20 @@ public class MediaContribution implements PublicationContext {
 
     @JsonProperty(PART_OF)
     private final SeriesEpisode partOf;
+    @JsonProperty(LINK)
+    private final URI link;
```
Comment (anetteOlli) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java:35]: Isn't this an associated artefact in reality?

Comment (anetteOlli) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java:35]: If the cristin post is a podcast and this links points to said podcast, the link is an associated artefact in my opinion.

Comment (truhacevkir) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java:35]: Fully possible. Do we place podcasts links, tv programs links in associatedArtifacts? If yes, i remove this pr.

Comment (truhacevkir) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java:35]: Do we use AssociatedArtifacts.associatedLink for that purpose? Isn't it mostly DOI? I will check with @brinxmat 

image

Comment (brinxmat) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java:35]: Associated artefact, yes.

Comment (truhacevkir) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java:35]: Fikk svar fra Rurik, du har rett. Fjerner pr.

**Pull request closed without merging**